documentation/docs/libraries/nodejs/exchange_guide.md

```diff
@@ -0,0 +1,238 @@
+- Secure seed management.
+- Account management (with multiple accounts and multiple addresses).
+- Confirmation monitoring.
+- Deposit address monitoring.
+- Backup and restore functionality.
+
+## How Does it Work?
+
+The Wallet Library is a stateful package with a standardized interface for developers to build applications involving value transactions. It offers abstractions to handle payments and can optionally interact with Stronghold for seed handling, seed storage, and state backup.
+
+For further reference, you can read our [wallet documentation here](https://wiki.iota.org/wallet.rs/welcome).
+
+The following examples cover the *multi-account approach* using the `NodeJS` binding:
+
+1. Set up the Wallet Library.
+2. Create an account for each user.
+3. Generate a User Address to Deposit Funds
+4. Listen to events.
+5. Check the user balance.
+6. Enable withdrawals.
+
+:::note
+
+If you are looking for other languages, please read the [wallet library overview](https://wiki.iota.org/wallet.rs/welcome).
+
+:::
+
+Since all `wallet.rs` bindings are based on core principles provided by the `wallet.rs` library, the outlined approach is very similar regardless of the programming language of your choice.
+
+### 1. Set up the Wallet Library
+First, you should install the components that are needed to use `wallet.rs` and the binding of your choice; it may vary a bit from language to language. In the case of the `NodeJs` binding, it is straightforward since it is distributed via the `npm` package manager. We also recommend you use `dotenv` for password management.
+
+You can read more about [backup and security in this guide](https://wiki.iota.org/introduction/guides/backup_security).
+
+```bash
+npm install @iota/wallet dotenv
+```
+
+#### 1.1 Generate a mnemonic
+
+<CodeBlock className="language-javascript">
+  {generate_mnemonic}
+</CodeBlock>
+
+Then, input your password to the `.env` file like this:
+
+```bash
+touch .env
+```
+
+```bash
+SH_PASSWORD="here is your super secure password"
+MNEMONIC="here is your super secure 24 word mnemonic, it's only needed here the first time"
+```
+
+Once you have everything needed to use the `wallet.rs` library, it is necessary to initialize the `AccountManager` instance with a secret manager(`Stronghold` by default) and client options.
+
+:::note
+
+Manage your password with the utmost care.
+
+:::
+
+By default the Stronghold file will be called `wallet.stronghold`. It will store the seed (derived from the mnemonic) that serves as a cryptographic key from which all accounts and related addresses are generated.
+
+One of the key principles behind the `stronghold` is that no one can get a seed out of it, so you should also backup the mnemonic (24 words) in a secure place, because there is no way to recover it from the `.stronghold` file. You deal with all the accounts purely via the `AccountManager` instance where all complexities are hidden under the hood and are dealt with securely.
```

Comment:
```suggestion
One of the key principles behind the `stronghold` is that no one can get a seed out of it, so you should also back up your 24-word mnemonic in a secure place because there is no way to recover it from the `.stronghold` file. You deal with accounts using the `AccountManager` instance exclusively, where all complexities are hidden under the hood and are dealt with securely.
```